src/contexts/mobile-menu.tsx

```diff
@@ -112,10 +112,12 @@ const MobileMenuProvider = ({ children }: MobileMenuProviderProps) => {
 
 		router.events.on('routeChangeComplete', handleRouteChange)
 		router.events.on('routeChangeError', handleRouteChange)
+		router.events.on('hashChangeComplete', handleRouteChange)
```

Comment:
Note: the `hashChangeComplete` addition here has been made to ensure that when an `#anchor-links` in the mobile menu is clicked, the menu closes. Upstream, this is not the case.

Reply:
This is a really nice interaction I think 💯